src/inject/dynamic-theme/modify-css.ts

```diff
@@ -303,7 +303,12 @@ function shouldIgnoreImage(selectorText: string, selectors: Array<string | RegEx
     const ruleSelectors = selectorText.split(/,\s*/g);
     for (let i = 0; i < selectors.length; i++) {
         const ignoredSelector = selectors[i];
-        if (ruleSelectors.some((s) => typeof ignoredSelector === 'string' ? s === ignoredSelector : ignoredSelector.test(s))) {
+        if (ruleSelectors.some((s) => {
+            if (ignoredSelector && ignoredSelector.constructor === RegExp) {
```

Comment:
The regular expression is not created anywhere, you will just have strings like `/abc/`. I suggest you run the code and put a breakpoint. Anyway, I suggest you to look into a simple string solution, as far as regular expressions in our config file will make it harder to maintain.